Gruntfile.js

```diff
@@ -196,9 +234,10 @@ module.exports = function( grunt ) {
 	grunt.loadNpmTasks( 'grunt-contrib-jasmine' );
 	grunt.loadNpmTasks( 'grunt-contrib-jshint' );
 
-	grunt.registerTask( 'scripts', [ 'browserify', 'jasmine', 'jshint' ] );
+	grunt.registerTask( 'scripts', [ 'browserify', 'jshint' ] );
 	grunt.registerTask( 'styles', [ 'sass', 'postcss' ] );
 	grunt.registerTask( 'default', [ 'scripts', 'styles' ] );
+	grunt.registerTask( 'test', [ 'jasmine' ] );
```

Comment:
Did you mean to keep this split out as a separate task? I don't think there's harm in just running it as a part of the `scripts` task.